packages/mui-types/package.json

```diff
@@ -23,9 +23,9 @@
   "scripts": {
     "build": "pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files",
     "build:node": "node ../../scripts/build.mjs node",
-    "build:stable": "node ../../scripts/build.mjs stable",
-    "build:types": "tsx ../../scripts/buildTypes.mts",
-    "build:copy-files": "node ../../scripts/copyFiles.mjs",
+    "build:stable": "MUI_EXPERIMENTAL_MJS=true node ../../scripts/build.mjs stable",
```

Comment:
I was thinking we could just set this in CI globally? Then test with the pkg-pr-new packages